docs/whats-new.rst

```diff
@@ -4,6 +4,12 @@ What's New
 v0.1.4 (? ? 2015)
 --------------------
 
+Added topography.
+
+Added automated linear stability analysis.
+
+Added multi layered model subclass. 
+
```

Comment:
Topography does not work in that semi-infinite model, but I had to make little changes just to be compatible since the topographic term is computes in the kernel.